data/ui/preferences.blp

```diff
@@ -0,0 +1,474 @@
+using Gtk 4.0;
+using Adw 1;
+
+template PreferencesWindow : Adw.PreferencesWindow {
+  can-navigate-back: true;
+
+  Adw.PreferencesPage {
+    icon-name: "applications-system-symbolic";
+    title: "General";
+
+    Adw.PreferencesGroup {
+      title: _("Import Settings");
+      description: _("Settings for importing graphs");
+
+      Adw.EntryRow import_delimiter {
+        title: _("Delimiter");
+        max-width-chars: 10;
+      }
+
+      Adw.ComboRow import_separator {
+        title: _("Decimal Separator");
+        subtitle: _("The character that is used as decimal separator");
+        model: StringList {
+          strings [".", ","]
+        };
+      }
+
+      Adw.ActionRow {
+        title: _("Column X");
+        subtitle: _("Which column to load for the X-data");
+
+        SpinButton column_x {
+          valign: center;
+          adjustment: column_spin;
+          numeric: true;
+          value: 0;
+          wrap: true;
+        }
+      }
+
+      Adw.ActionRow {
+        vexpand: false;
+        title: _("Column Y");
+        subtitle: _("Which column to load for the Y-data");
+
+        SpinButton column_y {
+          valign: center;
+          adjustment: column_spin;
```

Comment:
Please divide the column_spin up into a column_spin_x and column_spin_y.
In the current implementation they're coupled, so if you change the value of one it will also change the value of the other to that value,